socorro/unittest/app/test_socorro_app.py

```diff
@@ -22,8 +22,9 @@ def test_instantiation(self):
         with pytest.raises(NotImplementedError):
             sa.main()
 
+    @mock.patch('socorro.app.socorro_app.setup_crash_reporting')
```

Comment:
This mock avoids``sentry_sdk`` choking when treating a ``MagicMock`` as a DSN string.